Yosemite/Yosemite/Stores/TaxClassesStore.swift

```diff
@@ -0,0 +1,153 @@
+import Foundation
+import Networking
+import Storage
+
+
+// MARK: - TaxClassesStore
+//
+public class TaxClassesStore: Store {
```

Comment:
nit: generally we name a Yosemite store with the singular form of the entity (e.g. `Product` -> `ProductStore`), I'd suggest `TaxClassStore`